src/hotspot/os/posix/signals_posix.cpp

```diff
@@ -436,70 +432,34 @@ bool PosixSignals::chained_handler(int sig, siginfo_t* siginfo, void* context) {
 // Note that the VM will print warnings if it detects conflicting signal
 // handlers, unless invoked with the option "-XX:+AllowUserSignalHandlers".
 //
-
-#if defined(BSD)
-extern "C" JNIEXPORT int JVM_handle_bsd_signal(int signo, siginfo_t* siginfo,
-                                               void* ucontext,
-                                               int abort_if_unrecognized);
-#elif defined(AIX)
-extern "C" JNIEXPORT int JVM_handle_aix_signal(int signo, siginfo_t* siginfo,
-                                               void* ucontext,
-                                               int abort_if_unrecognized);
-#else
-extern "C" JNIEXPORT int JVM_handle_linux_signal(int signo, siginfo_t* siginfo,
-                                               void* ucontext,
-                                               int abort_if_unrecognized);
-#endif
-
-#if defined(AIX)
-
-// Set thread signal mask (for some reason on AIX sigthreadmask() seems
-// to be the thing to call; documentation is not terribly clear about whether
-// pthread_sigmask also works, and if it does, whether it does the same.
-bool set_thread_signal_mask(int how, const sigset_t* set, sigset_t* oset) {
```

Comment:
As remarked above, this will conflict with JDK-8252533, since I remove this function too. Can we leave this out please?

Reply:
Reverted.